modules/vectra.py

```diff
@@ -27,36 +27,45 @@ def __init__(self, response):
                 detail = response.content
         except Exception: 
             detail = response.content
-        body = 'Status code: {code} - {detail}'.format(code=str(response.status_code), detail=detail)
-        super().__init__(body)
+        self.message = 'Status code: {code} - {detail}'.format(code=str(response.status_code), detail=detail)
```

Comment:
Minor nit.  I don't believe you need to assign the message to self.message here.  The call to super.__init__(message) should set (overwrite) self.message.

```
message = ...
super().__init__(message)